service/matching/matching_engine_test.go

```diff
@@ -238,7 +239,11 @@ func (s *matchingEngineSuite) newPartitionManager(prtn tqid.Partition, config *C
 }
 
 func (s *matchingEngineSuite) TestAckManager() {
-	m := newAckManager(s.logger)
+	controller := gomock.NewController(s.T())
+	defer controller.Finish()
```

Comment:
there's a gomock controller set up in SetupTest, you shouldn't make a new controller